docs/stream.md

```diff
@@ -156,52 +156,54 @@ stream = Stream(
 
 ## Stream crashing
 
-If your stream `crashes` for any reason, the event consumption will stop meaning that non event will be consumed from the `topic`.
-As an end user you are responsable of deciding what to do. In future version approaches like `re-try`, `stream engine stops on stream crash` might be introduced.
+If your stream `crashes` for any reason the event consumption is stopped, meaning that non event will be consumed from the `topic`. However, it is possible to set three different `error policies` per stream:
 
-```python title="Crashing example"
-import aiorun
+- `StreamErrorPolicy.STOP`: Stop the `Stream` when an exception occurs. The exception is raised after the stream is properly stopped.
```

Comment:
I would modify this to specify that it's the default and then remove the note down.

```diff
- `StreamErrorPolicy.STOP`
+ `StreamErrorPolicy.STOP` (**default**)
```